dev-util/hyprwayland-scanner/hyprwayland-scanner-0.3.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2024 Gentoo Authors
```

Comment:
for new package, just start at current year, so it should be1 "Copyright 2024 Gentoo Authors"